README.md

```diff
@@ -40,90 +40,34 @@ An easy way to get the full Panoptes stack running (see `fig.yml` to dig into th
 
-#### Installation
+* [fig](http://fig.sh)
 
-1. Ensure your repo directory starts with a lowercase letter - you may need to move `/Panoptes` to `/panoptes`.
+#### Usage
 
-2. Prepare your fig development environment config files. You should only have to do this before the first boot. **Note:** The fig docker environment uses linked docker containers, so your Postgres and Zookeeper hosts urls need to refer to these containers.
+1. Run `rake configure FIG_RAKE=off`
 
-3. Copy all the `config/*.yml.hudson` files to `config/*.yml`. The default values should work out of the box.
+2. Run `fig up`
 
-4. Run `./scripts/fig/up_panoptes.sh`.
-    * On the first run it will build the docker containers, setup the database and install the development and test environment gems.
-    * **Note:** It will not start the rails server(s), and will exit after db migration.
-
-5. Run `./scripts/fig/up_panoptes.sh` again.
-  * On second run the script will start Panoptes and all the dependent services.
-  * **Note:** this script does not recreate containers to avoid installing gems and migrating the database.
-   * If you've added new gems, or you otherwise need to recreate the build image, run `fig up`.
-
-6. Run `scripts/fig/run_cmd_panoptes.sh "bundle install && rails runner db/fig_dev_seed_data/fig_dev_seed_data.rb"`
-    * This will seed the fig development database in the docker container.
-    * **Note:** Run this only after step 4 has completed successfully.
-
-7. Finally, if you want to apply schema migrations, run `scripts/fig/migrate_db_panoptes.sh`
+3. Open a new terminal and run `rake db:create db:migrate`
 
+4. Once step 3 is finished, run `rails runner db/fig_dev_seed_data/fig_dev_seed_data.rb`
+  * This will seed the development database with an Admin user and a Doorkeeper client application for API access.
 
 This will get you a working copy of the checked out code base. Keep your code up to date and rebuild the image if needed!
 
-Finally there are some helper scripts to get access to a console, bash shell etc. **Note:** these commands build a new run container
-* To get a rails console `scripts/fig/rails_console_panoptes.sh`
-  + **Note:** you can override the RAILS_ENV by passing a valid argument, just make sure you've setup the DB for it!
-* To get a bash console `scripts/fig/run_cmd_panoptes.sh bash`
-* You can also attach a bash process to the running container, e.g. `docker exec -it panoptes_panoptes_1 bash`
-  + Assuming the 'panoptes_panoptes_1' container is running, use `fig ps` or `docker ps` to check.
-
-**Note:** if you've ever built a Panoptes docker container before you should just run `fig up` instead of the `./scripts/fig/up_panoptes.sh` to ensure the previously built container is not re-used. After rebuilding you should be good to use `./scripts/fig/up_panoptes.sh` script to use the re-created containers.
+If you've added new gems you'll need to rebuild the docker image via the command in step 4.
```

Comment:
Step 4 doesn't reference a fig build step. Best to make and explicit reference here.